app/views/flexible_page/show.html.erb

```diff
@@ -7,6 +7,20 @@
   } %>
 <% end %>
 
+<% content_for :before_content do %>
+  <div class="govuk-width-container">
+    <%= render "govuk_publishing_components/components/breadcrumbs", {
+      collapse_on_mobile: true,
+      breadcrumbs: [
+        {
+          title: "Home",
+          url: "/",
```

Comment:
For MVP breadcrumbs this won't match what it should be for history pages (Home -> History of the UK Government) so should we either hard code that in here or include it in the YAML somehow? (again to note that I'm raising a problem on code I think I wrote).

Reply:
I've updated this to hardcode the history breadcrumb for now, since that's the scope of this scaffolding.